teacherchannel/module.py

```diff
@@ -0,0 +1,248 @@
+                            str(i.user_id),
+                        )
+                        for i in obj.teachers
+                    ]
+                )
+
+        items = [Item(i) for i in TeacherChannelDB.get_all(ctx.guild.id)]
+        if len(items) < 1:
+            await ctx.reply(_(ctx, "No teacher channels are set."))
+            return
+        table = utils.text.create_table(
+            items,
+            header={
+                "o_channel": _(ctx, "Original channel"),
+                "t_channel": _(ctx, "Teacher channel"),
+                "teachers": _(ctx, "Teachers"),
+            },
+        )
+        for page in table:
+            await ctx.send("```" + page + "```")
+
+    @commands.check(check.acl)
+    @teacherchannel_.command(name="set", aliases=["add"])
+    async def teacherchannel_set(
+        self, ctx, master: nextcord.TextChannel, slave: nextcord.TextChannel
+    ):
+        """<master channel> <slave channel>
+        Create a teacherchannel relation to sync."""
+        config = TeacherChannelDB.add_channel(ctx.guild.id, master.id, slave.id)
+        if not config:
+            await ctx.reply(_(ctx, "This channel is already assigned."))
+            return
+        await self._sync(slave, master, config)
+        await ctx.reply(_(ctx, "Teacher channel relation created."))
+        await guild_log.info(
+            ctx.author,
+            ctx.channel,
+            f"Teacher channel relation created. Master: <#{master.id}>, slave: <#{slave.id}>.",
+        )
+
+    @commands.check(check.acl)
+    @teacherchannel_.command(name="unset", aliases=["remove", "delete"])
+    async def teacherchannel_unset(self, ctx, channel: nextcord.TextChannel):
+        """<master channel or slave channel>
+        Remove teacherchannel relation."""
+        success = TeacherChannelDB.remove_channel(ctx.guild.id, channel.id)
+        if not success:
+            await ctx.reply(_(ctx, "This channel was never assigned."))
+            return
+        await ctx.reply(_(ctx, "Teacher channel unset."))
+        await guild_log.info(
+            ctx.author,
+            ctx.channel,
+            f"Teacher channel relation of channel {channel.name} unset.",
+        )
+
+    @commands.check(check.acl)
+    @teacherchannel_.group(name="teacher")
+    async def teacherchannel_teacher(self, ctx):
+        """**set** | **unset** <master channel or slave channel> <teacher member>
+        Manipulation with teachers of a teacher channel."""
+        await utils.discord.send_help(ctx)
+
+    @commands.check(check.acl)
+    @teacherchannel_teacher.command(name="set", aliases=["add"])
+    async def teacherchannel_teacher_set(
+        self, ctx, channel: nextcord.TextChannel, teacher: nextcord.Member
+    ):
+        """<master or slave channel> <teacher member>
+        Mark user as a teacher of a channel. Permission sync will not apply to this member. Grants permissions to the teacher channel (slave) only."""
```

Comment:
Another thing about your docstrings: Help command would display the `<master or slave channel> <teacher member>` instead of real human-friendly description of the function.